data/mods/ssb/moves.ts

```diff
@@ -110,6 +110,41 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Fire",
 	},
 
+	// cleann
+	maliciousintent: {
+		accuracy: 100,
+		basePower: 90,
+		category: "Physical",
+		desc: "Reveals a Pokemon from the other team.",
+		shortDesc: "Reveals a Pokemon from the other team.",
+		name: "Malicious Intent",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Shadow Bone', target);
+		},
+		self: {
```

Comment:
why is this in self?

Reply:
So that even if the opponent has a sub, it would still activate

Reply:
oh ok